centreon/tests/api/features/Notification.feature

```diff
@@ -486,3 +486,226 @@ Feature:
+          "events": 5,
+          "ids": [53,56],
+          "extra": {
+            "event_services": 2
+          }
+        },
+        {
+          "type": "servicegroup",
+          "events": 5,
+          "ids": [1,2]
+        }
+      ],
+      "messages": [
+        {
+          "channel": "Slack",
+          "subject": "Hello world !",
+          "message": "just a small message"
+        }
+      ],
+      "users": [20,21],
+      "is_activated": false
+    }
+    """
+    Then the response code should be "201"
+
+    When I send a PATCH request to '/api/latest/configuration/notifications/2' with body:
+    """
+    {
+      "is_activated": true
+    }
+    """
+    Then the response code should be "404"
+
+  Scenario: Disable a notification as a non-admin user with sufficient rights
+    Given the following CLAPI import data:
+    """
+    CONTACT;ADD;test-user;test-user;test-user@localservice.com;Centreon@2022;0;1;en_US;local
+    CONTACT;setparam;test-user;reach_api;1
+    ACLMENU;add;ACL Menu test;my alias
+    ACLMENU;GRANTRW;ACL Menu test;1;Configuration;Notifications;
+    ACLRESOURCE;add;ACL Resource test;my alias
+    ACLRESOURCE;grant_hostgroup;ACL Resource test;Linux-Servers
+    ACLRESOURCE;grant_hostgroup;ACL Resource test;Printers
+    ACLGROUP;add;ACL Group test;my alias
+    ACLGROUP;addmenu;ACL Group test;ACL Menu test
+    ACLGROUP;addresource;ACL Group test;ACL Resource test
+    ACLGROUP;addcontact;ACL Group test;test-user
+    SG;ADD;service-grp1;service-grp1-alias
+    ACLRESOURCE;grant_servicegroup;ACL Resource test;service-grp1
+    SG;ADD;service-grp2;service-grp2-alias
+    CONTACT;ADD;user-name1;user-alias1;user1@mail.com;Centreon!2021;0;0;;local
+    CONTACT;ADD;user-name2;user-alias2;user2@mail.com;Centreon!2021;0;0;;local
+    """
+    And I am logged in with "test-user"/"Centreon@2022"
+    And a feature flag "notification" of bitmask 2
+    When I send a POST request to '/api/latest/configuration/notifications' with body:
+      """
+      {
+        "name": "notification-name",
+        "timeperiod": 2,
+        "resources": [
+          {
+            "type": "hostgroup",
+            "events": 5,
+            "ids": [53,56],
+            "extra": {"event_services": 2}
+          },
+          {
+            "type": "servicegroup",
+            "events": 5,
+            "ids": [1]
+          }
+        ],
+        "messages": [
+          {
+            "channel": "Slack",
+            "subject": "Hello world !",
+            "message": "just a small message"
+          }
+        ],
+        "users": [20,21],
+        "is_activated": true
+      }
+      """
+    Then the response code should be "201"
+
+    When I send a PATCH request to '/api/latest/configuration/notifications/1' with body:
+    """
+    {
+      "is_activated": false
+    }
+    """
+    Then the response code should be "204"
+
+  Scenario: Enable a notification as a non-admin user with insufficient rights
```

Comment:
Just a side note for another time: try to reduce the number of scenarios to improve CI performances ( I try to keep it to one scenario per endpoint when possible)